js/coinsbit.js

```diff
@@ -0,0 +1,520 @@
+            const precision = {
+                'amount': this.safeInteger (market, 'stockPrec'),
+                'price': this.safeInteger (market, 'moneyPrec'),
+            };
+            const limits = {
+                'amount': {
+                    'min': this.safeFloat (market, 'minAmount'),
+                    'max': undefined,
+                },
+                'price': {
+                    'min': undefined,
+                    'max': undefined,
+                },
+                'cost': {
+                    'min': undefined,
+                    'max': undefined,
+                },
+            };
+            parsedMarketList.push ({
+                'id': id,
+                'symbol': symbol,
+                'base': base,
+                'quote': quote,
+                'baseId': baseId,
+                'quoteId': quoteId,
+                'active': isActive,
+                'precision': precision,
+                'limits': limits,
+                'info': market,
+            });
+        }
+        return parsedMarketList;
+    }
+
+    async fetchTicker (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const response = await this.publicGetTicker (this.extend ({ 'market': market['id'] }, params));
+        const ticker = this.safeValue (response, 'result');
+        return this.parseTicker (ticker, market);
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const response = await this.publicGetTickers (params);
+        const tickers = this.safeValue (response, 'result');
+        const tickersIds = Object.keys (tickers);
+        const parsedTickers = [];
+        for (let tickerIndex = 0; tickerIndex < tickersIds.length; tickerIndex++) {
+            const tickerId = tickersIds[tickerIndex];
+            const tickerObject = tickers[tickerId];
+            const ticker = this.safeValue (tickerObject, 'ticker');
+            ticker['at'] = this.safeValue (tickerObject, 'at');
+            parsedTickers.push (this.parseTicker (ticker, this.findMarket (tickerId)));
+        }
+        return this.filterByArray (parsedTickers, 'symbol', symbols);
+    }
+
+    parseTicker (ticker, marker) {
+        const timestamp = this.safeTimestamp (ticker, 'at', this.milliseconds ());
+        const datetime = this.iso8601 (timestamp);
+        const high = this.safeFloat (ticker, 'high');
+        const low = this.safeFloat (ticker, 'low');
+        const bid = this.safeFloat (ticker, 'bid');
+        const ask = this.safeFloat (ticker, 'ask');
+        const open = this.safeFloat (ticker, 'open');
+        const close = this.safeFloat (ticker, 'last');
+        const last = this.safeFloat (ticker, 'last');
+        const change = last - open;
+        const percentage = parseFloat (change / open) * parseFloat (100);
+        const average = parseFloat (last + open) / parseFloat (2);
```

Comment:
https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings